src/tests/Interop/Swift/SwiftInlineArray/SwiftInlineArray.swift

```diff
@@ -100,3 +100,14 @@ public func swiftFunc3(a0: F3) -> Int {
     hasher.combine(a0.element);
     return hasher.finalize()
 }
+
+@frozen public struct F4 {
+    public var elements: (UInt32, UInt32)
+}
```

Comment:
I'm curious. How does the mapping of the Swift `F4` struct to C# works?
```
[InlineArray(2)]
struct F4
{
    private static int _staticField1 = 1;
    private uint _element0;
    private static int _staticField2 = 2;
}
```
I would expect there to be a static field in the Swift `F4` struct so that we have static field on C# side as well. Am I missing something?

Reply:
In this test static fields are "c# only` addition to the struct, they are not visible to swift. The point of this test is to verify that adding static fields (which are not subject to struct lowering) does not affect lowering of the payload.

Reply:
I wonder why we even allow adding "extra" fields to projected structs from Swift. My thinking was that the C# structs that are used for Swift calls should be a projection of Swift counterparts (no more no less) and additional fields would be added to extra wrappers over the C# lowered struct.

However, I see the same was done for `record_struct_physical_lowering` so there is probably a good use-case for adding static fields that are not part of lowering...

Reply:
The use case here if I recall correctly was to keep the metadata pointer. A static field made sense as the metadata pointer belonged to the type rather than the instance.

Reply:
Make sense. Thanks for the insight!